src/pyhf/tensor/pytorch_backend.py

```diff
@@ -322,8 +324,11 @@ def normal_cdf(self, x, mu=0.0, sigma=1.0):
         Returns:
             PyTorch FloatTensor: The CDF
         """
-        normal = torch.distributions.Normal(mu, sigma)
-        return normal.cdf(x)
+        # the following variant is numerically more stable for low p-values
+        # than the default version that uses torch.erf
+        # erfc(x) = 1 - erf(x)
+        mu, sigma = broadcast_all(mu, sigma)
+        return 0.5 * torch.erfc(-((x - mu) * sigma.reciprocal() / math.sqrt(2)))
```

Comment:
Can you provide a citation for this calculation for reference?

Reply:
since erf(-x) = -erf(x) and erfc(x) := 1 - erf(x) we can replace 1 + erf(x) by erfc(-x). The rest is just copied from the [implementation of torch.Normal.cdf](https://github.com/pytorch/pytorch/blob/3bbedb34b9b316729a27e793d94488b574e1577a/torch/distributions/normal.py#L78-L81). I expanded a bit on that in the comment.